cli/cmd.js

```diff
@@ -67,79 +65,134 @@ const setup = function setup() {
       verify.verify(options);
     });
 
-
   commander
     .command('start')
     .option('-o, --org <org>', 'the organization')
     .option('-e, --env <env>', 'the environment')
     .option('-k, --key <key>', 'key for authenticating with Edge')
     .option('-s, --secret <secret>', 'secret for authenticating with Edge')
-    .option('-c, --cluster', 'will cluster the server')
     .option('-p, --processes <processes>', 'number of processes to start, defaults to # of cores')
     .option('-d, --pluginDir <pluginDir>','absolute path to plugin directory')
     .option('-r, --port <portNumber>','override port in the config.yaml file')
     .description('start the gateway based on configuration')
-    .action((options)=>{
+    .action((options)=> {
       options.error = optionError;
-      options.secret = options.secret || process.env.EDGEMICRO_SECRET ;
-      options.key =  options.key || process.env.EDGEMICRO_KEY;
+      options.secret = options.secret || process.env.EDGEMICRO_SECRET;
+      options.key = options.key || process.env.EDGEMICRO_KEY;
       options.org = options.org || process.env.EDGEMICRO_ORG;
       options.env = options.env || process.env.EDGEMICRO_ENV;
-      options.cluster =  options.cluster || process.env.EDGEMICRO_CLUSTER ;
-      options.processes =  options.processes || process.env.EDGEMICRO_PROCESSES;
+      options.processes = options.processes || process.env.EDGEMICRO_PROCESSES;
 
-      if(cluster.isMaster && options.port){
+      if (options.port) {
         portastic.test(options.port)
-          .then(function(isAvailable){
-            if(!isAvailable) {
+          .then(function (isAvailable) {
+            if (!isAvailable) {
               options.error('port is not available.');
               process.exit(1);
             }
-            
+
           });
       }
+
       if (!options.key ) {return  options.error('key is required');}
       if (!options.secret ) {return  options.error('secret is required');}
       if (!options.org ) { return  options.error('org is required'); }
       if (!options.env ) { return  options.error('env is required'); }
-      run.start(options,(err)=>{
-        if(cluster.isMaster){
-          if(!err) {
-            console.log("edgemicro started successfully.")
-          }else{
-            console.error(err);
-          }
-        }
-      });
+      run.start(options);
+    });
+
+  commander
+    .command('reload')
+    .option('-o, --org <org>', 'the organization')
+    .option('-e, --env <env>', 'the environment')
+    .option('-k, --key <key>', 'key for authenticating with Edge')
+    .option('-s, --secret <secret>', 'secret for authenticating with Edge')
+    .option('-p, --processes <processes>', 'number of processes to start, defaults to # of cores')
```

Comment:
@kevinswiber Its an optional attribute. If one needs to bring up a different number of workers (may be more or less than the original number of workers)  for any reasons, they can use this option. If left alone, original number of workers will be brought up.